src/compiler/diagnosticMessages.json

```diff
@@ -3733,5 +3733,15 @@
     "Extract constant": {
         "category": "Message",
         "code": 95006
+    },
+
+    "enclosing scope": {
+        "category": "Message",

```

Comment:
i think you need two versions of the full message. we can not guarantee that the translations of `Extract_to` + `eclosing_scope` will make sense in all languages.

Reply:
may be add three variations of the message:

`extract {0} to enclosing scope`
`extract {0} to {1} scope`
`extract {0} to {1}  // global / module`